### PR TITLE
python-isort - 4.3.4 - new package required for pylint

### PR DESCRIPTION
**Pull request by JPeterMugaas**

### CODE REVIEW

File: mingw-w64-python-isort/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+pkgname=("${MINGW_PACKAGE_PREFIX}-python2-${_realname}"  "${MINGW_PACKAGE_PREFIX}-python3-${_realname}")
+pkgver=4.3.4
+pkgrel=1
+pkgdesc="A Python utility / library to sort Python imports. (mingw-w64)"
+arch=('any')
+url='https://github.com/timothycrosley/isort'
+license=('MIT')
+makedepends=("${MINGW_PACKAGE_PREFIX}-python2"
+             "${MINGW_PACKAGE_PREFIX}-python3"
+             "${MINGW_PACKAGE_PREFIX}-python2-futures"
```
Comment (Alexpux) [inline mingw-w64-python-isort/PKGBUILD]: Seems it must be "future" not "futures"

File: mingw-w64-python-isort/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  for _f in "${pkgdir}${MINGW_PREFIX}"/bin/*.py; do
+    sed -e "s|/usr/bin/env |${MINGW_PREFIX}|g" \
+        -e "s|${PREFIX_WIN}|${MINGW_PREFIX}|g" -i ${_f}
+  done
+#### end section ####
+}
+
+package_python2-isort() {
+  depends=("${MINGW_PACKAGE_PREFIX}-python2"
+           "${MINGW_PACKAGE_PREFIX}-python2-futures")
```
Comment (Alexpux) [inline mingw-w64-python-isort/PKGBUILD]: The same

**Comment by Alexpux** [comment]
@JPeterMugaas  you forgot to commit isort install files

**Comment by Alexpux** [comment]
@JPeterMugaas ping

**Comment by JPeterMugaas** [comment]
https://github.com/Alexpux/MINGW-packages/pull/4637

**Pull request merged**